.github/workflows/main.yml

```diff
@@ -0,0 +1,125 @@
+on:
```

Comment:
Can you please rename the file to `ci.yml`? We already have too many `main.yml` everywhere (yep, ansible). On vscode-ansible extension the workflow name is [ci]()https://github.com/ansible/vscode-ansible/tree/main/.github/workflows) which is short, helping with display of checks tables. 

I do not want to enforce the "ci" name and if the team prefer another name we can also change it in both places. Considering that the pipelines inside are likely to be have names like `test`, `build`, `package`, i think that "ci" is ok. On python projects we use "tox", but that does not apply to js projects.

Screenshot with one version (length matters):
![](https://sbarnea.com/ss/Screen-Shot-2021-09-16-09-09-46.30.png)

Screenshot with current version (scrolling is needed on both axes but possible only on one):
![](https://sbarnea.com/ss/Screen-Shot-2021-09-16-09-11-42.14.png)

Reply:
ok